apps/Launcher/syncwidget.cpp

```diff
@@ -225,8 +225,11 @@ void SyncWidget::setSceneFiles(QMap<QString, QString> sceneFiles) {
         const QString& sceneName = keys[i];
 
         QCheckBox* checkbox = new QCheckBox(sceneName);
-        checkbox->setChecked(true);
-
+        checkbox->setChecked(false);
+        QString defaultName = "default";
+        if(QString::compare(defaultName, sceneName, Qt::CaseInsensitive)==0){
```

Comment:
Ah, very nice.  I didn't know that the operator was overlayed, and that does look much better.